src/dialog/dialog.component.ts

```diff
@@ -101,9 +108,12 @@ export class Dialog implements OnInit, AfterViewInit, OnDestroy {
 		this.placement = this.dialogConfig.placement.split(",")[0];
 		this.data = this.dialogConfig.data;
 
-		this.resizeSubscription = Dialog.resizeObservable.subscribe(() => {
-			this.placeDialog();
-		});
+		// fallback if elementService isn't available
+		if (!this.elementService) {
+			this.resizeSubscription = Dialog.resizeObservable.subscribe(() => {
+				this.placeDialog();
+			});
```

Comment:
Why not do:
```js
this.resizeSubscription = Dialog.resizeObservable.subscribe(this.placeDialog);
```

Reply:
I don't _think_ we'll run into scoping issues with that ... I'll give it a try 👍